tsqc/pages/mutations.py

```diff
@@ -78,25 +77,76 @@ def make_muts_panel(log_y, tsm):
         }
     )
     trees_hist = hv.DynamicMap(selected_hist(bp_df), streams=streams)
+    trees_hist.opts(
+        width=config.PLOT_WIDTH,
+        height=100,
+        hooks=[customise_ticks],
+        xlabel="tree density",
+    )
     trees_line = hv.Segments(bp_df, ["position", "y0", "x1", "y1"])
+    trees_line.opts(
+        width=config.PLOT_WIDTH,
+        height=100,
+        yaxis=None,
+        ylabel=None,
+        xlabel="tree breakpoints",
+        color="grey",
+        line_width=0.5,
+        alpha=0.5,
+    )
+
+    if config.GENES_FILE:
+        genes_df = tsm.genes_df(config.GENES_FILE)
+        annot_track = make_annotation_plot(tsm, genes_df)
+        layout = (
```

Comment:
It would be nice to avoid laying out the shared elements twice. would something like this work?
```python
layout = (main << time_hist << site_hist) + trees_hist + trees_line
if config.GENES_FILE is not None:
      genes_df = tsm.genes_df(config.GENES_FILE)
      annot_track = make_annotation_plot(tsm, genes_df)
      layout += annot_track
return pn.Column(layout.opts(shared_axes=True).cols(1))
```